publisher/server_only/README.md

```diff
@@ -1,8 +1,8 @@
 # Server-Only UID2 Integration Example
 
-[This example](https://example-srvonly-integ.uidapi.com/) demonstrates how a content publisher can use the [UID2 services](https://github.com/UnifiedID2/uid2docs/tree/main) to implement the [custom UID2 integration workflow](https://unifiedid.com/docs/guides/custom-publisher-integration).
+[This example](https://example-srvonly-integ.uidapi.com/) demonstrates how a content publisher can use the [UID2 services](https://unifiedid.com/docs/intro) to implement the [custom UID2 integration workflow](https://unifiedid.com/docs/guides/custom-publisher-integration).
 
-For an example application using the [Client-Side Identity JavaScript SDK](https://github.com/UnifiedID2/uid2docs/blob/main/api/v2/sdks/client-side-identity.md), see [UID2 SDK Integration Example](../standard/README.md).
+For an example application using the [Client-Side Identity JavaScript SDK](https://unifiedid.com/docs/sdks/client-side-identity), see [UID2 SDK Integration Example](../standard/README.md).
```

Comment:
For an example application using the [Client-Side JavaScript SDK](https://unifiedid.com/docs/sdks/client-side-identity), see [UID2 SDK Integration Example](../standard/README.md).